boa_engine/Cargo.toml

```diff
@@ -14,6 +14,15 @@ readme = "../README.md"
 [features]
 profiler = ["boa_profiler/profiler"]
 deser = ["boa_interner/serde"]
+intl = [
+    "dep:icu_locale_canonicalizer",
```

Comment:
I think this syntax was introduced fairly recently, we will probably need to increase the MSRV.